.github/workflows/driver-adapter-smoke-tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        adapter: ['neon:ws', 'neon:http', planetscale, pg, libsql]
+        adapter: ['neon:ws', 'neon:http', planetscale, pg, libsql, tidbcloud]
 
```

Comment:
You also need to add the new env var to lines 48+ in this file, otherwise the secret is not provided by GitHub Actions and tests fail.

Reply:
Miss the env, I have added it now.